synapsis/src/test/java/com/soen/synapsis/unit/appuser/AppUserDetailsServiceTest.java

```diff
@@ -34,13 +34,16 @@ void tearDown() throws Exception {
     @Test
     void loadUserByUsername() {
         String email = "joeusertest@mail.com";
+        when(appUserRepository.findByEmail(email)).thenReturn(new AppUser("joe", "12345678", "joe@mail.com", Role.CANDIDATE));
```

Comment:
If we create the dummy user, this test will be more accurate as well

Reply:
This test actually considers the standard authentication flow. In either case, the sample users created by data.sql do not impact the test code.